src/test/shared/settingsConfiguration.test.ts

```diff
@@ -75,4 +77,125 @@ describe('DefaultSettingsConfiguration', function () {
             })
         })
     })
+    // Methods 'disable' and 'shouldDisplayPrompt' write to a specific setting and cannot use the same setting as used for testing 'readSetting' and 'writeSetting'
+    describe('disable', function () {
+        const promptName = 'promptName'
+        const fakePromptSetting = { promptName: false }
+        const fakePromptSettingTrue = { promptName: true }
+
+        beforeEach(async function () {
+            sandbox = sinon.createSandbox()
```

Comment:
why use sinon? the pattern shown on lines 20-30 should work for this setting also. Just use a different SETTINGS_KEY.

Reply:
Friendly reminder to everyone that refactoring tests to reduce copy-paste code _is_ important for maintainability. Your test code should not be an afterthought.